### DISCUSSION TITLE
Is this is Totally free?

### DISCUSSION THREAD

**Discussion by arhamsarwar786** [faizqadri234/fastlane-with-github-actions-implementation#14] [Q&A] [answered]
I need to know that is I can use Fastlane for Flutter apps and website deployments. Does Fastlane use YAML Language for configuration? Whats the Exact Source for Learning?

**Comment by faizqadri234** [comment, marked as answer]
1. Yes fastlane is completely free. It has no additional charges.
2. Yes you can use fastlane to automate release of your flutter apps. And sorry you cannot use it for your website deployments. But you can use GitHub actions for this purpose. Or if you really want to use some tools then many tools available for setting up a CI/CD pipeline for web development including Jenkins, GitLab CI/CD, CircleCI, and Travis CI etc.
3. Fastlane itself doesn't use YAML language. We write all the script in ruby.  And then we integrate fastlane command in YAML language to make it CD (continuous deployment)
4. You can learn everything about fastlane from its official [Documentation](https://docs.fastlane.tools/)

Hope I answered it well 😊. Thanks

**Reply by arhamsarwar786** [reply]
OK, It's good to know but do you mean I do not use Fastlane for my web apps deployment like react-js app or flutter web app?
